freqtrade/strategy/interface.py

```diff
@@ -548,6 +548,23 @@ def version(self) -> Optional[str]:
         """
         return None
 
+    def populate_any_indicators(self, basepair: str, pair: str, df: DataFrame, tf: str,
+                                informative: DataFrame = None, coin: str = "",
+                                set_generalized_indicators: bool = False) -> DataFrame:
+        """
+        Function designed to automatically generate, name and merge features
+        from user indicated timeframes in the configuration file. User can add
+        additional features here, but must follow the naming convention.
+        Defined in IStrategy because Freqai needs to know it exists.
```

Comment:
I'd add a comment here that says that this method is ONLY used / called if freqAI is enabled.
That could replace the "defined here because" ... sentence.